src/main/java/uk/gov/digital/ho/hocs/casework/api/StageTagsDecoratorImpl.java

```diff
@@ -46,7 +47,7 @@ private Boolean hasDataFieldAndValue(Stage stage, String field, String value) {
     }
 
     private Boolean addHomeSecReplyTag(Stage stage) {
-        if(hasDataFieldAndValue(stage, HOME_SEC_REPLY_FIELD_NAME, "TRUE") || hasDataFieldAndValue(stage, OVERRIDE_PO_TEAM_NAME, "Home Secretary")) {
+        if(hasDataFieldAndValue(stage, HOME_SEC_REPLY_FIELD_NAME, "TRUE") || hasDataFieldAndValue(stage, OVERRIDE_PO_TEAM_NAME, "Home Secretary") || hasDataFieldAndValue(stage, PO_TEAM_NAME, "Home Secretary")) {
```

Comment:
Is there a test we could write for this?

Reply:
Good shout, added some in